Resources/Prototypes/_Nuclear14/Entities/Mobs/NPCs/pets.yml

```diff
@@ -0,0 +1,18 @@
+#MARK: Haggis
+- type: entity
+  name: Haggis
+  id: N14MobCatHaggis
+  parent: MobCat
+  description: A little rough around the edges, but still cute as can be.
+  components:
+  - type: Sprite
+  - sprite: _Nuclear14\Mobs\Pets\cathaggis.rsi
```

Comment:
You added a - here thats not needed

Reply:
oh I went to go fix it myself before seeing you'd already pushed an update, you're too kind as always Peppy.